### DISCUSSION TITLE
Block texture mapping like MS-DOS version

### DISCUSSION THREAD

**Discussion by ed-kurlyak** [LostArtefacts/TRX#1849] [General]
I'm interested in how to fix the source code so that there is a cubic display of textures - without antialiasing, as it was in the game MS-DOS. So that the texture was without antialiasing - cubes, blocks look.

I tried to change GFX_3D_Renderer_Init() so (changed shader samplers) but nothing in result, same presented antialiasing:

`

GFX_GL_Sampler_Bind(&renderer->sampler, 0);
	//GFX_GL_Sampler_Parameterf(
	  //  &renderer->sampler, GL_TEXTURE_MAX_ANISOTROPY_EXT, 16.0f);

	GFX_GL_Sampler_Parameteri(
		&renderer->sampler, GL_TEXTURE_MIN_FILTER, GL_NEAREST);

	GFX_GL_Sampler_Parameteri(
		&renderer->sampler, GL_TEXTURE_MAG_FILTER, GL_NEAREST);

	GFX_GL_Sampler_Parameterf(
    &renderer->sampler, GL_TEXTURE_MAX_ANISOTROPY_EXT, 1.0f);

    GFX_GL_Program_Init(&renderer->program);
    GFX_GL_Program_AttachShader(
        &renderer->program, GL_VERTEX_SHADER, "shaders\\3d.vsh");
    GFX_GL_Program_AttachShader(
        &renderer->program, GL_FRAGMENT_SHADER, "shaders\\3d.fsh");
    GFX_GL_Program_Link(&renderer->program);

`

**Comment by Gerwin2k** [comment]
I think, what you mean is already a setting in the game.
Start the game - open the detail level options - then the second option in the list: Filtering 
Set it ftom bilinear to off.
